manifests/profiles/openshift-ambient.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      proxyMetadata:
+        ISTIO_META_ENABLE_HBONE: "true"
+
+  components:
+    cni:
+      enabled: true
+      namespace: kube-system
+    ztunnel:
+      enabled: true
+      namespace: kube-system
+    ingressGateways:
+    - name: istio-ingressgateway
+      enabled: false
+
+  values:
+    global:
+      platform: openshift
+    cni:
+      ambient:
+        enabled: true
+        redirectMode: "inpod"
+      cniBinDir: /var/lib/cni/bin
+      cniConfDir: /etc/cni/multus/net.d
+      chained: false
+      cniConfFileName: "istio-cni.conf"
+      excludeNamespaces:
+      - kube-system
+      logLevel: info
+      privileged: true
+      provider: "multus"
+    istio_cni:
+      enabled: true
+      chained: false
+    pilot:
+      variant: distroless
+      env:
+        # Setup more secure default that is off in 'default' only for backwards compatibility
+        VERIFY_CERTIFICATE_AT_CLIENT: "true"
+        ENABLE_AUTO_SNI: "true"
+
+        PILOT_ENABLE_HBONE: "true"
+        CA_TRUSTED_NODE_ACCOUNTS: "istio-system/ztunnel,kube-system/ztunnel"
+        PILOT_ENABLE_AMBIENT_CONTROLLERS: "true"
+    ztunnel:
+      variant: distroless
+      redirectMode: "inpod"
+      seLinuxOptions:
```

Comment:
The other option is to use priviledged mode for ztunnel pods. 

Do we have `spc_t` profile on all the OCP deployments?